src/api/app/models/staging_workflow.rb

```diff
@@ -1,4 +1,14 @@
 class StagingWorkflow < ApplicationRecord
   belongs_to :project, inverse_of: :staging
   has_many :staging_projects, class_name: 'Project', inverse_of: :staging_workflow, dependent: :nullify
+
+  has_many :staged_requests, class_name: 'BsRequest', through: :staging_projects
+
+  def unassigned_requests
+    BsRequest.by_actions_target_projects(project.id) - staged_requests - ignored_requests
```

Comment:
[nitpick] Is it not possible to chain / combine it to a single query instead of doing three queries and do some calculations in Ruby afterwards? This seems not to be very efficient ...

Reply:
@ChrisBr:

> [nitpick] Is it not possible to chain / combine it to a single query instead of doing three queries and do some calculations in Ruby afterwards? This seems not to be very efficient ...

Sure, I agree. What about doing that later, after we also have the `ignored_requests` query? It is what I will work on after this is finished.

Reply:
Agreed :+1: